user-info-api/README.md

```diff
@@ -0,0 +1,84 @@
+# API definition for users metadata
```

Comment:
Can you reference the READMEs of the `user-info-*` projects in the parent README?
Best case summarize the three in one Markdown? :thinking:

Reply:
done